Description: No need of Makefile{,in,am} under debian/
 Upstream execute make under debian/ , which we don't need.
---
Origin: Laszlo Boszormenyi (GCS) <gcs@debian.hu>
Forwarded: not-needed
Last-Update: Wed Oct 19, 2011

--- syslog-ng-3.3.1.orig/Makefile.am
+++ syslog-ng-3.3.1/Makefile.am
@@ -1,4 +1,4 @@
-SUBDIRS = lib modules syslog-ng scripts tests doc contrib scl debian tgz2build build
+SUBDIRS = lib modules syslog-ng scripts tests doc contrib scl tgz2build build
 
 ACLOCAL_AMFLAGS = -I m4 --install
 
--- syslog-ng-3.3.1.orig/configure.in
+++ syslog-ng-3.3.1/configure.in
@@ -1159,7 +1159,6 @@ AC_OUTPUT(dist.conf
 	  contrib/Makefile
 	  scl/Makefile
 	  tests/Makefile 
-	  debian/Makefile
 	  tgz2build/Makefile
           tests/unit/Makefile
           tests/functional/Makefile
